experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -86,6 +96,108 @@ export type LoggerProviderConfig = {
   logRecordProcessors: LogRecordProcessor[];
 };
 
+/**
+ * @Returns param value, if set else returns the default value
+ */
+function getValueInMillis(envName: string, defaultValue: number): number {
+  return parseInt(process.env[envName] || '') || defaultValue;
+}
+
+/**
+ *
+ * @returns MetricReader[] if appropriate environment variables are configured
+ */
+function configureMetricProviderFromEnv(): MetricReader[] {
+  const metricReaders: MetricReader[] = [];
+  const metricsExporterList = process.env.OTEL_METRICS_EXPORTER?.trim();
+  if (!metricsExporterList) {
+    return metricReaders;
+  }
+  const enabledExporters = filterBlanksAndNulls(metricsExporterList.split(','));
+
+  if (enabledExporters.length === 0) {
+    diag.info('OTEL_METRICS_EXPORTER is empty. Using default otlp exporter.');
+  }
+
+  if (enabledExporters.includes('none')) {
+    diag.info(
+      `OTEL_METRICS_EXPORTER contains "none". Metric provider will not be initialized.`
+    );
+    return metricReaders;
+  } else {
```

Comment:
we return in the `if` block above so we can reduce nesting here by dropping the else.

Reply:
agree